cnd/src/http_api/orderbook.rs

```diff
@@ -298,8 +306,10 @@ mod tests {
         let json = r#"
         {
             "sell_token_contract": "0xB97048628DB6B661D4C2aA833e95Dbe1A905B280",
+            "bitcoin_ledger: regtest"
             "buy_quantity": "300",
             "sell_quantity": "200",
+            "ethereum_ledger: "
```

Comment:
That is not valid JSON.

Reply:
How'd that get in there? Its hard to get good help :) Thanks for flagging this for me.

Reply:
This is now
```
        let json = r#"
        {
            "sell_token_contract": "0xB97048628DB6B661D4C2aA833e95Dbe1A905B280",
            "bitcoin_ledger": "regtest",
            "buy_quantity": "300",
            "sell_quantity": "200",
            "ethereum_ledger": {"chain_id":2},
            "absolute_expiry": 600,
            "refund_identity": "1F1tAaz5x1HUXrCNLbtMDqcw6o5GNn4xqX",
            "redeem_identity": "0x00a329c0648769a73afac7f9381e08fb43dbea72",
            "maker_addr": "/ip4/127.0.0.1/tcp/39331"
        }"#;
```